bootloader/Vagrantfile

```diff
@@ -109,6 +158,29 @@ def build_bootloader(boxname)
   EOF
 end
 
+def build_bootloader_target_win32(boxname)
+  return <<-EOF
+    cd /vagrant/bootloader
+    rm -rf ../PyInstaller/bootloader/Windows/*
+    python ./waf all CC=i686-w64-mingw32-gcc
+    python ./waf all CC=x86_64-w64-mingw32-gcc
+  EOF
+end
+
+def build_bootloader_target_osx(boxname)
+  return <<-EOF
+    cd /vagrant/bootloader
+    rm -rf ../PyInstaller/bootloader/Darwin/*
+    if [ ! -d /vagrant/osxcross ] ; then
+        mkdir -p /vagrant/osxcross
+        tar -C /vagrant/osxcross -xjf /vagrant/macOSX/osxcross.tar.bz2
+    fi
+    PATH=/vagrant/osxcross/bin/:$PATH
+    python ./waf all CC=x86_64-apple-darwin15-clang
+    python ./waf all CC=i386-apple-darwin15-clang
```

Comment:
I don't know whether 32 bit on OS X ist still used, but we have a [relate comment](https://github.com/htgoebel/pyinstaller/blob/a601f62d7076e73727c40a57e60e0a95ff2aa359/bootloader/wscript#L245) in th wscript, too.